configuration_test.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+import sys
+import os
+import re
+import subprocess
+import warnings
+from gslab_scons import _exception_classes
+
+def configuration_test(ARGUMENTS, gslab_python_version):
+    # Determines whether to print traceback messages
+    debug = ARGUMENTS.get('debug', False) 
+    if not debug:
+        # Hide traceback for configuration test only
+        # http://stackoverflow.com/questions/27674602/hide-traceback-unless-a-debug-flag-is-set
+        sys.tracebacklimit = 0 
+
+    # Checks initial prerequisites
+    try:
+        from gslab_scons import configuration_tests as config
+    except ImportError:
+        message = 'Your gslab_tools Python modules installation is outdated'
+        raise Exception(message)
+
+    config.check_python(gslab_python_version = gslab_python_version, 
+                        packages = ["yaml", "gslab_scons", "gslab_fill"])
+    config.check_lyx()
+    config.check_lfs()
+    stata_flavor = config.check_stata(["yaml"])
+
+    # Uncomment if using
+    # config.check_r(packages = ["yaml"]) 
+
+    # Loads arguments and configurations
+    mode = ARGUMENTS.get('mode', 'develop') # Gets mode; defaults to 'develop'
+
+    # Checks mode/version
+    if not (mode in ['develop', 'cache']):
+        message = "Error: %s is not a defined mode" % mode
+        raise _exception_classes.PrerequisiteError(message)
+    
+    # Get return list
+    if mode == 'cache':
+        cache_dir   = config.load_yaml_value("user-config.yaml", "cache")
```

Comment:
No, I think lines 160 and 188 in that file address your concern. Let me know if you disagree.